elephant/spade.py

```diff
@@ -289,37 +291,27 @@ def spade(data, binsize, winlen, min_spikes=2, min_occ=2, max_spikes=None,
         rank = comm.Get_rank()  # get rank of current MPI task
     else:
         rank = 0
-    output = {}
-    # Decide whether compute pvalue spectrum
-    if n_surr > 0:
-        # Compute pvalue spectrum
-        time_pvalue_spectrum = time.time()
-        pv_spec = pvalue_spectrum(data, binsize, winlen, dither=dither,
-                                  n_surr=n_surr, min_spikes=min_spikes,
-                                  min_occ=min_occ, max_spikes=max_spikes,
-                                  max_occ=max_occ, min_neu=min_neu,
-                                  spectrum=spectrum)
-        time_pvalue_spectrum = time.time() - time_pvalue_spectrum
-        print("Time for pvalue spectrum computation: {}".format(
-            time_pvalue_spectrum))
-        # Storing pvalue spectrum
-        output['pvalue_spectrum'] = pv_spec
-    elif 0 < alpha < 1:
-        warnings.warn('0<alpha<1 but p-value spectrum has not been '
-                      'computed (n_surr==0)')
+
+    if output_format not in ['concepts', 'patterns']:
+        raise ValueError(
+            "The output_format value has to be one between"
```

Comment:
'one between' is confusing
change to `has to be 'x', 'y', 'z',...` or `has to be one of ...`